cmd/zz_gen_cmd_dnshelp.go

```diff
@@ -219,7 +219,9 @@ func displayDNSHelp(name string) error {
 		ew.writeln(`Credentials:`)
 		ew.writeln(`	- "CF_API_EMAIL":	Account email`)
 		ew.writeln(`	- "CF_API_KEY":	API key`)
-		ew.writeln(`	- "CLOUDFLARE_API_KEY":	Alias to CLOUDFLARE_API_KEY`)
+		ew.writeln(`	- "CF_API_TOKEN":	API token`)
+		ew.writeln(`	- "CLOUDFLARE_API_KEY":	Alias to CF_API_KEY`)
+		ew.writeln(`	- "CLOUDFLARE_API_TOKEN":	Alias to CF_API_TOKEN`)
 		ew.writeln(`	- "CLOUDFLARE_EMAIL":	Alias to CF_API_EMAIL`)
```

Comment:
Nit: These don't line up for me, presumably because tabs have variable width. To ensure alignment, maybe spaces would be preferred. (Not a showstopper IMO, just wanted to point this out)

Reply:
it's generated code, the PR need to be rebased